lib/chef/knife/ssl_fetch.rb

```diff
@@ -28,6 +28,8 @@ class SslFetch < Chef::Knife
         require 'socket'
         require 'uri'
         require 'openssl'
+        require_relative 'proxified_socket'
```

Comment:
require_relative is an anti-pattern because it bypasses the Ruby "don't require a file multiple times" check.